editor/src/clj/editor/scene.clj

```diff
@@ -738,6 +749,30 @@
   (assert (= (count buf) (* picking-drawable-size picking-drawable-size)) "picking buf of unexpected size")
   (map (partial aget buf) picking-buf-spiral-indices))
 
+(g/defnk produce-overlay-anchor-pane-props [scene ^:try tool-info-text]
+  (if-let [error (:error scene)]
+    {:children [{:fx/type cljfx.fx.text-area/lifecycle
+                 :anchor-pane/bottom 0
+                 :anchor-pane/left 0
+                 :anchor-pane/right 0
+                 :anchor-pane/top 0
+                 :style-class "info-text-area"
+                 :editable false
+                 :wrap-text true
+                 :text (string/join \newline (error-message-lines [error]))}]}
+    (if-let [overlay-anchor-pane-props (:overlay-anchor-pane-props scene)]
+      overlay-anchor-pane-props
+      (let [info-text (or tool-info-text (:info-text scene))]
```

Comment:
Since you use `^:try`, `tool-info-text` here could be an `ErrorValue`. But it seems that the code expects it to be a string. Is it because `tool-info-text` is an error value only when `scene` has `:error`?

Reply:
Well, the code will only use it as a string if it is a string. If it is an `ErrorValue`, we won't render anything here.

However, I think we should fall back on the `:info-text` from the `scene` if the `tool-info-text` is an `ErrorValue` rather than showing nothing. I'll fix!